src/explicitode.jl

```diff
@@ -373,7 +369,7 @@ function taylorinteg{T<:Number}(f, q0::Array{T,1}, t0::T, tmax::T,
         end
     end
 
-    return view(tv,1:nsteps), view(xv,:,1:nsteps)' #for xv, first do view, then transpose (otherwise it crashes)
+    return view(tv,1:nsteps), view(transpose(xv),1:nsteps,:)
```

Comment:
Did you check the effect (on the benchmarks) of this change? I suspect that `transpose` allocates an array, whereas the view does not.